index.js

```diff
@@ -14,4 +14,150 @@ menuItem.forEach((menuItem) => {
+    image: './img/Dbg2.png',
+    stack: ['FACEBOOK', 'Full Stack Dev', '2015'],
+    technologies: ['HTML', 'CSS', 'JavaScript', 'Ruby', 'Bootstrap'],
+    liveLink: '',
+    sourceLink: '',
+  },
+  {
+    name: 'Facebook 360',
+    description: 'Lorem Ipsum is simply dummy text of the prinhe 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent.',
+    image: './img/Dbg3.png',
+    stack: ['FACEBOOK', 'Full Stack Dev', '2015'],
+    technologies: ['HTML', 'CSS', 'JavaScript', 'Ruby', 'Bootstrap'],
+    liveLink: '',
+    sourceLink: '',
+  },
+  {
+    name: 'UBER Navigation',
+    description: 'Lorem Ipsum is simply dummy text of the prinhe 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent.',
+    image: './img/Dbg4.png',
+    stack: ['FACEBOOK', 'Full Stack Dev', '2015'],
+    technologies: ['HTML', 'CSS', 'JavaScript', 'Ruby', 'Bootstrap'],
+    liveLink: '',
+    sourceLink: '',
+  },
+];
+
+const cardsContainer = document.querySelector('#About');
+
+projects.forEach((project) => {
+  const card = document.createElement('div');
+  card.classList.add('about-section');
+
+  card.innerHTML = `
+    <img src="${project.image}" class="card-img" />
+      <div class="canopy">
+        <h4 class="about-head">${project.name}</h4>
+        <ul class="cano">
+          <li class="tec">FACEBOOK</li>
+          <li class="tech">Full Stack Dev</li>
+          <li class="tech">2015</li>
+        </ul>
+        <p class="canopy-txt">
+        A daily selection of privately personalized reads; no accounts or
+          sign-ups required
+        </p>
+        <ul class="tech1">
+          <li class="tech-t">HTML</li>
+          <li class="tech-r">Ruby On Rails</li>
+          <li class="tech-t">CSS</li>
+          <li class="tech-j">JavaScript</li>
+        </ul>
+        
+        <button type="button" class="project-btn">See Project</button>
+      </div>
+  `;
+
+  cardsContainer.appendChild(card);
+
+  const button = card.querySelector('.project-btn');
+
+  const modalContainer = document.createElement('div');
+  modalContainer.classList.add('modal');
+  modalContainer.id = 'modal';
+
+  modalContainer.innerHTML = `
+  <div class="modal-content">
+    <span class="close">&times;</span>
+    <h2 class="project-title" id="projectTitle"></h2>
+    <ul id="cano">
+    </ul>
+    <div class="project-details">
+      <img class="project-image" id="projectImage" src="" alt="">
+      <div class="p-cont">
+      <p class="project-description" id="projectDescription"></p>
+      <div class="p-div">
+      <ul class="project-technologies" id="projectTechnologies"></ul>
```

Comment:
- [ ] Well! This list is okay, but there are still some adjustments needed in the margin, padding, and font styles. 🙏 Please closely follow the design and update your code accordingly.

| Your | Professional |
| --- | --- |
| ![Group 68 (2)](https://github.com/Danny-imo/my-project/assets/55840999/7b311cae-3d2a-4e67-83ef-88d690de1175) | ![Group 65 (2)](https://github.com/Danny-imo/my-project/assets/55840999/3819b872-1abd-4243-97db-93f40b3702f7) |


By making these adjustments, you will ensure that the list matches the design more closely in terms of margin and font styles. Good luck with your updates! ✨🎨

Reply:
- [ ] Note the space between the windows popup technologies on the Figma design, it is smaller than the space on your design. Kindly check it to follow the project requirement.
### Your design :point_down: 
![](https://user-images.githubusercontent.com/55840999/248082531-7b311cae-3d2a-4e67-83ef-88d690de1175.png)
### What it's supposed to look like :point_down: 
![](https://user-images.githubusercontent.com/55840999/248082508-3819b872-1abd-4243-97db-93f40b3702f7.png)